src/ui.tsx

```diff
@@ -93,6 +99,9 @@ const styles = () => ({
     paddingTop: "1px",
     width: "61px",
     marginRight: "9px",
```

Comment:
If you're doing flexbox do you need the margin?

Reply:
It appears that the margin right are for adding space between the cards. Removing it resulted in this....
<img width="294" alt="Screenshot 2021-08-13 at 16 05 42" src="https://user-images.githubusercontent.com/37402968/129377870-ed19587e-3a80-4f5e-b819-be2f7ac1fd8a.png">


We can keep the margin right rule on the children element or we can add a width to the parent element and then do justify space around..
<img width="1788" alt="Screenshot 2021-08-13 at 16 16 39" src="https://user-images.githubusercontent.com/37402968/129381274-4d39c7ce-761a-470d-b06f-fca36c0b55a2.png">

Reply:
Okay - stick with it then :-)